Taskfile.yml

```diff
@@ -456,6 +457,41 @@ tasks:
     cmds:
      - "{{.SCRIPTS_ROOT}}/generate-helm-manifest.sh {{.KUBE_RBAC_PROXY}} {{.LOCAL_REGISTRY_CONTROLLER_DOCKER_IMAGE}} {{.PUBLIC_REGISTRY}} {{.LATEST_VERSION_TAG}} `pwd`/{{.CONTROLLER_ROOT}}"
 
+  controller:install-helm:
```

Comment:
You can (and should) remove this line: `--set azureClientSecret=$AZURE_CLIENT_SECRET \`. It's not needed because later you run `controller:make-aadpodidentity-secret` which makes the secret simulating managed identity. If you don't delete this line we might have access to both secrets and I am not sure what happens then. We want to make sure we're using the AAD Pod Identity path. Deleting the `azureClientSecret` should still allow everything to pass.